<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Files Renamer by Folco</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="boxFolders">
      <property name="title">
       <string>Folders</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="3">
        <widget class="QPushButton" name="buttonDestination">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QCheckBox" name="checkDeleteSource">
         <property name="text">
          <string>Delete the source files after a successful operation</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="editSource">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="buttonSource">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QCheckBox" name="checkOverwrite">
         <property name="text">
          <string>Overwrite existing files</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="editDestination">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkDestination">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labelDestination">
         <property name="text">
          <string>Destination:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelSource">
         <property name="text">
          <string>Source:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="boxDecoration">
      <property name="title">
       <string>Name decoration</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <spacer name="hspacerDecoration">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelParentFolder">
         <property name="text">
          <string>Use some parent folders:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editSeparator">
         <property name="text">
          <string> - </string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelSeparator">
         <property name="text">
          <string>Separator between the folders:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelPrefix">
         <property name="text">
          <string>Use a prefix:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinParentFolder">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editPrefix">
         <property name="text">
          <string>Page </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="boxFiles">
      <property name="title">
       <string>Files list</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelDoubleClick">
         <property name="text">
          <string>Double-click an item to open a preview</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableFiles">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>120</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Current name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New name</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hlayoutFilesButtons">
         <item>
          <widget class="QPushButton" name="buttonSelectAll">
           <property name="text">
            <string>Select all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonUnselectAll">
           <property name="text">
            <string>Unselect all</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonRename">
      <property name="text">
       <string>Rename selection</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
